datalink/hoverdrive.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<VOTABLE xmlns="http://www.ivoa.net/xml/VOTable/v1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2">
+
+  <RESOURCE type="meta" utype="adhoc:service" name="ColumnDocumentationRedirect">
+    <DESCRIPTION>
+      Redirect to the most relevant documentation link for a column.
+    </DESCRIPTION>
+    <PARAM name="accessURL" datatype="char" arraysize="*"
+          value="$baseUrl$/api/hoverdrive/column-docs-redirect"/>
+    <GROUP name="inputParams">
+      <PARAM name="table" datatype="char" use="required">
```

Comment:
I think we cannot have the 'use' attribute in the PARAM so I suspect we will have to remove it from the three PARAMs

Reply:
Also it looks as if the PARAMs need a value attribute. I think we can set value="" for these which are being templated.